h/static/scripts/test/streamer-test.js

```diff
@@ -74,41 +75,39 @@ describe('streamer', function () {
       websocketUrl: 'ws://example.com/ws',
     };
 
-    streamer = proxyquire('../streamer', {
+    Streamer = proxyquire('../streamer', {
       './websocket': FakeSocket,
     });
   });
 
   it('should not create a websocket connection if websocketUrl is not provided', function () {
     fakeSettings = {};
     createDefaultStreamer();
+    activeStreamer.connect();
```

Comment:
Good point. I decided to keep the auto-connect behaviour last minute on this PR, and didn't revert the test changes.